cmd/version/version.go

```diff
@@ -1,6 +1,26 @@
 package version
 
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
 var (
 	DefaultCloudURLStr = "https://cloud-api.calyptia.com"
 	Version            = "dev" // To be injected at build time:  -ldflags="-X 'github.com/calyptia/cli/cmd/version.Version=xxx'"
 )
+
+func NewVersionCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:           "version",
+		Short:         "v",
```

Comment:
```suggestion
		Aliases:        []string{"v"},
```

"Short" means a short description, not an alias.

Reply:
Actually no need for short hand "v" using full subcommand name is enough.